lib/core/Blockchain.ts

```diff
@@ -210,19 +210,17 @@ export default class Blockchain implements IBlockchain {
   }
 
   public async getWriterValueTimeLock (): Promise<ValueTimeLockModel | undefined> {
-
     const response = await this.fetch(this.writerLockUri);
-    const responseBodyString = await ReadableStream.readAll(response.body);
```

Comment:
worth adding another negative test where the response.body is not readable or is not JSON string. Before the change, it will throw other exceptions. After the change, it no longer throws that exception since the reading/parsing logic moved to later place.

Reply:
Sync offline.